app/models/on_the_fly_calc.rb

```diff
@@ -47,15 +47,21 @@ def compute_avg_and_ranges_hash
   end
 end
 
-private
+public
 
 # Get all of the questions asked during peer review for the given team's work
 def peer_review_questions_for_team(team, round_number = nil)
-  topic_id = SignedUpTeam.find_by(team_id: team.id).topic_id
-  review_questionnaire_id = review_questionnaire_id(round_number, topic_id)
-  Question.where(questionnaire_id: review_questionnaire_id)
+  if !team.nil? and !SignedUpTeam.find_by(team_id: team.id).nil?
+    topic_id = SignedUpTeam.find_by(team_id: team.id).topic_id
+    review_questionnaire_id = review_questionnaire_id(round_number, topic_id)
+    Question.where(questionnaire_id: review_questionnaire_id)
+  else
+    nil
```

Comment:
Ruby methods return 'false' by default, is the else nil part is required?

Reply:
code change was made